{{ cookiecutter.package_name }}/pyproject.toml

```diff
@@ -27,6 +27,28 @@ dependencies = [
 ]
 dynamic = ["version"]
 
+[project.urls]
+{%- if cookiecutter.project_url %}
+Homepage = "{{ cookiecutter.project_url }}"
+{% else %}
+Homepage = "https://sunpy.org"
+{%- endif %}
+{% if cookiecutter.download_url %}
+Download = "{{ cookiecutter.download_url }}"
+{% endif -%}
+{% if cookiecutter.sourcecode_url %}
```

Comment:
I think we are going to need to figure out a way to know what the github repo (I.e. sunpy/sunpy) is. We could parse it out of this variable maybe?

I also suggest we don't make this optional for the reason we will need it for other things like towncrier etc.